<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QMainWindow" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Abandoned house</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>1080</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>1922</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1922</width>
      <height>1080</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1922</width>
      <height>1080</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="l_name_game">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>671</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(72, 74, 89, 0);
color: #660000;
font: 40pt &quot;Nosifer&quot;;
letter-spacing: -4px;
word-spacing: 9px;</string>
     </property>
     <property name="text">
      <string>Abandoned house</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-50</y>
       <width>1922</width>
       <height>1180</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(25, 25, 25);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="p_start">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>551</width>
       <height>101</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(72, 74, 89, 0);
color: rgb(0, 0, 0);
letter-spacing: -1px;
word-spacing: 2px;
font: 60pt &quot;Zombie Holocaust&quot;;
border:none;

</string>
     </property>
     <property name="text">
      <string>START A NEW GAME</string>
     </property>
    </widget>
    <widget class="QPushButton" name="p_load">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>340</y>
       <width>321</width>
       <height>101</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(72, 74, 89, 0);
color: rgb(0, 0, 0);
letter-spacing: -1px;
word-spacing: 2px;
font: 60pt &quot;Zombie Holocaust&quot;;
border:none;

</string>
     </property>
     <property name="text">
      <string>LOAD GAME</string>
     </property>
    </widget>
    <widget class="QPushButton" name="p_about">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>490</y>
       <width>371</width>
       <height>91</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(72, 74, 89, 0);
color: rgb(0, 0, 0);
letter-spacing: -1px;
word-spacing: 2px;
font: 60pt &quot;Zombie Holocaust&quot;;
border:none;

</string>
     </property>
     <property name="text">
      <string>About game</string>
     </property>
    </widget>
    <widget class="QPushButton" name="p_exit">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>610</y>
       <width>211</width>
       <height>151</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(72, 74, 89, 0);
color: rgb(0, 0, 0);
letter-spacing: -1px;
word-spacing: 2px;
font: 60pt &quot;Zombie Holocaust&quot;;
border:none;

</string>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <zorder>graphicsView</zorder>
    <zorder>l_name_game</zorder>
    <zorder>p_start</zorder>
    <zorder>p_load</zorder>
    <zorder>p_about</zorder>
    <zorder>p_exit</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
